wasmcloud-test-util/src/provider_test.rs

```diff
@@ -280,12 +280,11 @@ pub async fn start_provider_test(config: TomlMap) -> Result<Provider, anyhow::Er
     // set logging level for capability provider with the "RUST_LOG" environment variable,
     // default level is "info"
     let log_level = match config.get("rust_log") {
-        Some(TomlValue::String(level)) if level.parse::<log::Level>().is_ok() => level.to_string(),
-        None => "info".to_string(),
-        Some(x) => {
-            eprintln!("invalid 'rust_log' setting '{}', using 'info'", x);
-            "info".to_string()
+        Some(TomlValue::String(level)) => {
```

Comment:
per your PR comment, I think you still want the `if level.parse::<log::Level>().is_ok()` here

Reply:
okay I'm fine with that, especially since this is the test util and it may only affect a log level for a test suite run and not a provider in production